mLRS/Common/hal/esp-rxclock.h

```diff
@@ -44,7 +43,7 @@ IRQHANDLER(
 #if defined(ESP8266)
 IRAM_ATTR void CLOCK_IRQHandler(void)
 #elif defined(ESP32)
-IRAM_ATTR bool CLOCK_IRQHandler(void * timerNo)
+IRAM_ATTR void CLOCK_IRQHandler()
```

Comment:
if it can be (void) the #ifdef disecting between esp8266 and esp32 is obsolete, isn't it?

Reply:
Good point, addressed.